project/Publish.scala

```diff
@@ -31,32 +32,7 @@ object Publish extends AutoPlugin {
   override def projectSettings = Seq(
     bintrayOrganization := Some("akka"),
     bintrayPackage := "akka-http",
-    bintrayRepository := (if (isSnapshot.value) "snapshots" else "maven")
-  )
-}
-
-object DeployRsync extends AutoPlugin {
-  import scala.sys.process._
-  import sbt.complete.DefaultParsers._
-
-  override def requires = plugins.JvmPlugin
-
-  trait Keys {
-    val deployRsyncArtifact = taskKey[Seq[(File, String)]]("File or directory and a path to deploy to")
-    val deployRsync = inputKey[Unit]("Deploy using SCP")
-  }
-
-  object autoImport extends Keys
-  import autoImport._
-
-  override def projectSettings = Seq(
-    deployRsync := {
-      val (_, host) = (Space ~ StringBasic).parsed
-      deployRsyncArtifact.value.foreach {
-        case (from, to) =>
-          val result = Seq("rsync", "-rvz", s"$from/", s"$host:$to").!
-          require(result == 0, "rsync should return success")
-      }
-    }
+    bintrayRepository := (if (isSnapshot.value) "snapshots" else "maven"),
+    publishRsyncHost in ThisBuild := "akkarepo@gustav.akka.io"
```

Comment:
Does that really belong here? Maybe move it to the `docs` module in `build.sbt`.